src/Umbraco.Web.UI.Client/src/views/languages/overview.html

```diff
@@ -38,7 +38,7 @@
                 <tbody>
                     <tr ng-repeat="language in vm.languages | orderBy:'name' track by language.id" ng-click="vm.editLanguage(language)" style="cursor: pointer;">
                         <td>
-                            <i class="icon-globe" style="color: #BBBABF; margin-right: 5px;"></i>
+                           <umb-icon icon="icon-globe" class="language-icon" aria-hidden="true"></umb-icon>
                             <span>{{ language.name }}</span>
```

Comment:
@poornimanayar it doesn't need `aria-hidden` here since the component itself takes care of this 🙂👍

Reply:
Thanks @bjarnef :) I have corrected it :)